gensim/models/tfidfmodel.py

```diff
@@ -301,10 +301,12 @@ def __init__(self, corpus=None, id2word=None, dictionary=None, wlocal=utils.iden
             Only when `pivot` is not None will pivoted document length normalization be applied.
             Otherwise, regular TfIdf is used.
         slope : float, optional
-            Parameter required by pivoted document length normalization which determines the slope to which
-            the `old normalization` can be tilted. This parameter only works when `pivot` is defined.
+            It is the parameter required by pivoted document length normalization which determines the slope to which
```

Comment:
Regression (PR contained had a newer, fixed version).

Out of curiosity -- what was this conflict from? This branch was already up-to-date with `develop`.